thumbor/loaders/file_fallback_http_loader.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# thumbor imaging service
+# https://github.com/thumbor/thumbor/wiki
+
+# Licensed under the MIT license:
+# http://www.opensource.org/licenses/mit-license
+# Copyright (c) 2011 globo.com timehome@corp.globo.com
+
+import re
+import tornado.httpclient
+from . import LoaderResult
+from thumbor.utils import logger
+from datetime import datetime
+from os import fstat
+from os.path import join, exists, abspath
+from urllib import unquote
+from tornado.concurrent import return_future
+from urlparse import urlparse
+from functools import partial
+
+@return_future
+def load(context, path, callback):
+    file_path = join(context.config.FILE_LOADER_ROOT_PATH.rstrip('/'), unquote(path).lstrip('/'))
+    file_path = abspath(file_path)
+    inside_root_path = file_path.startswith(context.config.FILE_LOADER_ROOT_PATH)
+
+    result = LoaderResult()
+
+    if inside_root_path and exists(file_path):
```

Comment:
Here in the if block you could call the `file_loader.load` method instead of copy/pasting it here.